spec/fixtures/swift/default/Shop.swift

```diff
@@ -0,0 +1,22 @@
+/* DO NOT EDIT | Generated by gyro */
+
+import RealmSwift
+import Foundation
+
+final class Shop: Object {
+
+  enum Attributes: String {
+    case active = "active"
+    case isOpened = "isOpened"
+    case isValid = "isValid"
+    case name = "name"
+    case promo = "promo"
+  }
+
+  dynamic var active: Bool = true
+  dynamic var isOpened: Bool = false
+  dynamic var isValid: Bool = false
+  dynamic var name: String = ""
```

Comment:
Would be nice to have a test case with a non empty default string too, to ensure both are covered

Reply:
Wait sorry I reviewed too quickly I though you fixed the case for strings already and not just int and bool, ok.

Would still be nice to handle default values for Strings properly too and not limit that fix to int and bool I guess.

Reply:
I agree with you. Thanks for feedback.